docs/chart-lint-publish-readme.md

```diff
@@ -0,0 +1,99 @@
+# chart-lint-publish
+
+## Purpose
+This workflow is designed to perform the following tasks:
+1. Validate Helm charts in pull requests by checking for changes in the Helm chart directory.
+2. Generate a list of charts that are modified in the base branch compared to the pull request branch.
+3. Optionally include or exclude specific charts from validation.
+4. Publish Helm charts to a Helm chart repository on successful validation.
+
+## Inputs
```

Comment:
Please leave a sentence space and give an introductory line for the section on Input..